doc/sneak.txt

```diff
@@ -353,7 +357,7 @@ g:sneak#s_next = 0
         persists until you move the cursor.
         Note: You can still use |;| and |,| as usual.
 
-g:sneak#absolute_dir = 0
+*g:sneak#absolute_dir* = 0
```

Comment:
I intentionally didn't do this because (specially for new users) it makes copy-pasting the code to one's vimrc more difficult.